plugins/CoreHome/javascripts/popover.js

```diff
@@ -142,7 +142,17 @@ var Piwik_Popover = (function () {
 
         /** Set the title of the popover */
         setTitle: function (titleHtml) {
+            var titleText = $('<div>' + titleHtml + '</div>').text();
+            if (titleText.length > 60) {
+                titleHtml = $('<span>').attr('class', 'tooltip').attr('title', titleText).html(titleHtml);
```

Comment:
Just tried it using  `titleHtml='"%<b>stt&7ä'` and it converts it to `"%stt&7ä` so some information gets lost. I wonder if it was possible to use `piwikHelper.escape(piwikHelper.htmlEntities(titleHtml))`. Eg `piwikHelper.escape(piwikHelper.htmlEntities('"%<b>stt&7ä'))` would result in `&#34;%&lt;b&gt;stt&amp;7&#228;` but not sure re downsides.

Otherwise just FYI: `piwikHelper.htmlDecode()` does some as above with `.text()`